frontend/constants/paths.ts

```diff
@@ -52,6 +68,11 @@ export const APIS = {
       title: '視聴ステータスのフェッチ',
     },
   },
+  USER_PROFILE: {
+    UPDATE: {
+      path: (userId: string | undefined) => `${backendUrl}/user/${userId}`,
+    },
```

Comment:
他の実装に沿うなら
```suggestion
    title: "ユーザのプロフィール更新"
```

とかを追加しておきたいです。

Reply:
追記しました！
次回以降も記述するようにします。